README.md

```diff
@@ -34,11 +34,13 @@ Ezyfox Server's benchmark involved broadcasting messages to 1000 concurrent user
 
 # Introduction
 
+A Free and Open Solution for Multiplayer Game Development
```

Comment:
I don't think ezyfox-server is just for game dev. We can build any real-time stuffs with it.